Source/WebCore/platform/graphics/gstreamer/eme/WebKitOpenCDMDecryptorGStreamer.cpp

```diff
@@ -201,4 +210,20 @@ static gboolean webKitMediaOpenCDMDecryptorDecrypt(WebKitMediaCommonEncryptionDe
     return returnValue;
 }
 
+static gchar* getContentType(GstElement* element)
```

Comment:
This function, unless I missed something is the same for the common class. Either make it public in the common decryptor header or move it to the GStreamerEMEUtilities file. I think the utilities file is better.

And it needs a more descriptive name is needed like getOriginalMediaTypeFromGstElement.